app/components/documents/document-preview/document-preview-modal.js

```diff
@@ -3,21 +3,17 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
 export default class DocumentsDocumentPreviewDocumentPreviewModal extends Component {
-  @tracked selectedVersion;
+  /**
+   * @argument piece
```

Comment:
Is er een specifieke reden om af te stappen van de lokale tracked property `@selectedVersion`? Zoals @ValenberghsSven al aanhaalde wordt het switchen tussen document versies hierdoor minder smooth, omdat er een volledige route herladen wordt. Dit geeft het flickering effect door de loading template die er tussenkomt, maar de gebruiker wordt bvb. ook teruggestuurd naar de 'Detail' tab terwijl hij op de 'Versies' tab zat. Lijkt mij een minder aangename gebruikerservaring vergeleken met voordien?

Reply:
De transitie is niet smooth en kan zo idd eigenlijk niet passeren. Het leek me wel conceptueel eenvoudiger van de selectie van versie op route-niveau te houden. Dat je een document-versie op zich kan bekijken óf door op een blokje in de versie-historiek van een ander document te klikken (op dat moment zijn route/url state en lokale state tegenstrijdig) leek me onnodig complex. Ik herbekijk 't.

Reply:
Reverted naar oud systeem https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1422/commits/970bf5d36461f24cbbe839c81afaec5f369af722